---
layout: page
title: Research Projects
image: /img/cottonBoll.png
---

<div>
<p align="center">
  <a href="#">seed development</a> |
  <a href="#">Link 2</a> |
  <a href="#">Link 3</a>
  <br><br>
</p>
</div>


## Evolutionary dynamics of cottonseed co-expression gene network
![](/research/seedNet.phenotype.jpg)

Cotton is best known as an important source of textile fibers; however, cotton seeds also provides vegetable oil and proteins, accounting for 10-15% of its total crop values. During the course of seed development in domesticated form of *Gossypium hirsutum* (TM1), oil content accumulates rapidly from 20 day to 30 days after flowering, and reaches approximately 25% of total seed weight. This profile is more similar to its D-genome than A-genome progenitor. Variance can also be seen from wild and domesticated species. **To gain insight into phenotypic diversification in cotton seeds, we conducted co-expression network analysis of developing seeds from diploid and allopolyploid cotton species and explored network properties.** Two different conceptual approaches were explored to help
understand gene coexpression network conservation and divergence at various evolutionary timescales and across ploidy levels. First, a multispecies coexpression network was constructed to provide a global view of coexpression network topology among species. Second, individual coexpression networks were constructed and compared for all species included in the study.

### Identify key multispecies network modules associated with devlopmental dynamics and phenotypic diversification  
![](/research/seedNet.multi1.jpg)
![](/research/seedNet.multi2.jpg)

### Compare individual species networks to reveal topological changes in response to evolutionary events  
Network comparisons among species indicate that topologies are altered in addition to gene expression profiles, indicating that changes in transcriptomic coexpression relationships play a role in the developmental architecture of cotton seed development. The global network topology of allopolyploids, especially fordomesticated *G. hirsutum*, resembles the network of the A-genome diploid more than that of the D-genome parent, despite its D-like phenotype in oil content. Among allopolyploids, intermodular relationships are more preserved between two different wild allopolyploid species than they are between wild and domesticated forms of a cultivated cotton, and regulatory connections of oil synthesis-related pathways are denser andmore closely clustered in domesticated vs. wild *G. hirsutum*. These results demonstrate **substantial modification of genic coexpression under domestication**.  
![](/research/seedNet.indiv.jpg)  
> Consensus modules represented by nodes are connected by edges representing intermodular relationship (highly correlated with Pearson’s correlation r>0.9) in each species. Node size is proportional to the total number of genes assigned to each consensus module. Node coloring reveals module eigengenes that are highly correlated (red) or anti-correlated (blue) with developmental stage. The preservation test statistic D is presented for between-genome comparisons. The closer D is to 1, the higher the preservation.

### Oil Synthesis and Lipid Related Gene Network Altered by Domestication
Regulatory connections of oil synthesis-related pathways are denser and more closely clustered in domesticated vs. wild *G. hirsutum*, suggesting **domestication strengthens the regulatory connections of oil synthesis and lipid related gene families**.  
![](/research/seedNet.oilNet.jpg)  
> Visualization of the oil related gene co-expression network in wild (Yuc; left panels) and domesticated (TM1; right panels) *G. hirsutum* developing seeds. The network overview (bottom panels) and a closer view (upper panels) of the “up-regulated” cluster are shown for each genome. The bottomtable summarizes the sub-network properties of density and clustering coefficient at the gene family level. A bootstrap approachwas used to test differences between TM1 and Yuc parameters, and * denotes high values with significance (P<0.05). TAG, triacylglycerol.


----
## Homoeolog gene co-expression network implicates biased recruitment of regulatory architecture of seed development  
Instead of using the aggregated expression of homoeolog gene pairs, we now constructed allopolyploid co-expression networks based on homoeolog-specific expressions. This approach allows direct comparisons of sub-genome networks against their corresponding diploid parental networks. In addition to revealing topological modificantions of the "redundant" duplicated seed network following allopolyploidization, this work aims to explore the assymetric evolution of parental regulatory connections underlying phenotypic evolution and domestication.

Stay tuned!



